README.md

```diff
@@ -1,39 +1,33 @@
-# {SimPlatform} connector
```

Comment:
Instead of {SimPlatform} and {SampleName} we should use uppercase SIM_PLATFORM and SAMPLE_NAME so that:
1. We are consistent with other doc topics that use uppercase in this fashion for items that should be replaced.
2. The template can be built and run in its original form even if the strings aren't replaced.

Reply:
Now renamed